pubsub/google/cloud/pubsub/_gax.py

```diff
@@ -148,7 +148,7 @@ def topic_delete(self, topic_path):
                 raise NotFound(topic_path)
             raise
 
-    def topic_publish(self, topic_path, messages):
+    def topic_publish(self, topic_path, messages, timeout=30):
```

Comment:
Sure, I just used the same default as [`google.gax.CallOptions`](https://github.com/googleapis/gax-python/blob/895fefd8aae2ee627fa22b7e623ededbee7f2e24/google/gax/__init__.py#L70).